src/data/code-samples/en/input-motion/swift-ui.md

```diff
@@ -0,0 +1,52 @@
+# Motion input - SwiftUI
+
+In SwiftUI, there is no built-in method to detect when a user shakes their device. However, you can easily create this functionality yourself by overriding the [`motionEnded`](https://developer.apple.com/documentation/uikit/uiresponder/1621090-motionended) method in UIWindow and setting up a custom notification.
```

Comment:
Remove `easily` because it doesn't look so straightfoward in SwiftUI ;) There is no SwiftUI `Window` equivalent for UIWindow?

Reply:
@JJdeGroot 
SwiftUI development has gotten better, but before it was a constant fight to bridge back with UIKit 😆. Still not ideal though.

[Window](https://developer.apple.com/documentation/swiftui/window) exists, just not for the iOS (macOS 13.0+ only). There is `WindowGroup`, but looks like it is for a different purpose.